preppal-be/routes/userApi.ts

```diff
@@ -1,14 +1,16 @@
-const {check, validationResult } = require("express-validator") ;
-const express = require("express");
-const router = express.Router();
+const expressUserApi = require("express");
+const routerUserApi = expressUserApi.Router();
+const jwtUserApi = require("jsonwebtoken");
+const configUserApi = require("../configs/secrets.ts")
 
-const User = require('../models/user.ts');
+const User = require("../models/user.ts");
 
-// router.get('/', (req, res) => res.send('user api testing!'));
-
-router.get("/", async (req, res) => {
+/**
+ * GET - Get all accounts
+ */
+routerUserApi.get("/", async (req, res) => {
     try {
-        const users = await User.find();
+        const users = await User.find().select("-password");
```

Comment:
what does this exactly do? Just curious

Reply:
this removes the password field in the lookup